Cargo.toml

```diff
@@ -24,3 +24,5 @@ log = "0.4.28"
 serde = { version = "1.0.228", features = ["derive"] }
 serde_json = "1.0.141"
 tokio = { version = "1.47.1", features = ["macros", "rt-multi-thread"] }
+uuid = { version = "1.0", features = ["v4"] }
```

Comment:
nit: it's good that you add this here, but then also set the import in register-server to workspace